### ISSUE TITLE
First load won't get skipped if first pull got an error

### ISSUE THREAD

**Issue by kr4ssi** [filipedeschamps/rss-feed-emitter#213]
To reproduce:
-Cut of internet
-Start your application
-Turn internet on
-Next pull will emit every item

**Comment by TobiTenno** [comment]
yeah, that's not really surprising, as it's not the first load if the first load errored...

**Comment by TobiTenno** [comment]
I'll see what I can do to have it check if the first load errored so it doesn't bypass it. Do you have the specific error it's throwing on our end?

**Comment by TobiTenno** [comment]
I've added a gitter for the repo so it's easier to find/talk to me

**Comment by kr4ssi** [comment]
> I'll see what I can do to have it check if the first load errored so it doesn't bypass it. Do you have the specific error it's throwing on our end?

It only says "Cannot connect to ..."

**Comment by TobiTenno** [comment]
thanks, that's all i needed.
